asynq/scheduler.py

```diff
@@ -92,7 +92,7 @@ def _execute(self, root_task):
             if len(self._tasks) > _debug_options.MAX_TASK_STACK_SIZE:
                 self.reset()
                 debug.dump(self)
-                raise RuntimeError('maximum task stack size exceeded while calling an asynq task')
+                raise RuntimeError('Number of scheduled tasks exceeded maximum threshold.')
```

Comment:
Would it make sense to add something like "while executing {}".format(root_task)?